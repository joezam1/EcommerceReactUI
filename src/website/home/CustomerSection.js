import React from 'react';


export default function CustomerSection(){
    return (<div className="customer-section">
              <input type="textbox" className="search-textbox"/>
              <button type="button" className="search-button">Find</button>
              <div className="contact-email">contact@flowershop.com</div>

              <div className="cart-info">CART: 0</div>
          
              <ul className="access-area-ul">
                  <li className="access-area-li"><button type="button" className="login bton">Login</button> </li>
                  <li className="access-area-li"><button type="button" className="register bton">Register</button></li>
              </ul>
            </div>
    )
}