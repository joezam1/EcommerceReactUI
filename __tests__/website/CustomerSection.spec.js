import React from 'react';
import {render, cleanup, waitFor, fireEvent} from '@testing-library/react';

import CustomerSection from '../../src/website/home/CustomerSection.js';

xdescribe('File CustomerSection', function(){
    afterEach(cleanup);
    test('take a snapshot', function(){
        let container = render(<CustomerSection/>);
        expect(container.asFragment(<CustomerSection/>)).toMatchSnapshot();
    })
})