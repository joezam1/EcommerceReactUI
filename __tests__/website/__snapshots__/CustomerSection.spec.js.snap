// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File CustomerSection take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="customer-section"
  >
    <input
      class="search-textbox"
      type="textbox"
    />
    <button
      class="search-button"
      type="button"
    >
      Find
    </button>
    <div
      class="contact-email"
    >
      contact@flowershop.com
    </div>
    <div
      class="cart-info"
    >
      CART: 0
    </div>
    <ul
      class="access-area-ul"
    >
      <li
        class="access-area-li"
      >
        <button
          class="login bton"
          type="button"
        >
          Login
        </button>
         
      </li>
      <li
        class="access-area-li"
      >
        <button
          class="register bton"
          type="button"
        >
          Register
        </button>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
